tasks/st1105_primary_server_port_check.rb

```diff
@@ -32,7 +32,10 @@ def port_test(dest, port)
   true
 end
 
+deprecation_msg = "This task is deprecated with support script uploading to be handled according to documentation, and will be removed in a future release.
+                   Please see this module's README for more information"
 results_json = []
+deprecation = [deprecation: deprecation_msg]
```

Comment:
Im not sure i would create a deprecation_msg variable here, i would also look how the array and hash is declared in result as your formatting is all wrong here